core.php

```diff
@@ -103,6 +103,11 @@
 	require_once( $g_config_path . 'config_inc.php' );
 }
 
+# Ensure we always have a consistent current working directory.
+# IIS with FastCGI seems to set CWD to directory where php.exe resides (see #33906).
+global $g_absolute_path;
+chdir( $g_absolute_path );
```

Comment:
Looks at first sight like a workaround for issues that might be caused by a PHP 8.2 bug on Windows + IIS + FastCGI.
We shouldn't rely at all on CWD, e.g. a plugin could change it.
As a side note: This would be the first time we use chdir in code.

Reply:
> might be caused by a PHP 8.2 bug on Windows + IIS + FastCGI

No idea if that's a PHP bug or not, I didn't search for it. It is a corner case anyway and not worth my time to analyze all the way through.

> We shouldn't rely at all on CWD, e.g. a plugin could change it.

We actually don't rely on CWD anymore, since the original fixes for [#33906](https://mantisbt.org/bugs/view.php?id=33906) I committed in March.

Setting CWD in core.php is actually not strictly necessary with the current code base as we now only use absolute path references. Consider that a "belt and braces" approach, trying to reduce the risk of this error occurring again. 

I thought it made sense for the MantisBT root to consistently be set as CWD, but if you disagree and don't want the chdir(), I'm fine with reverting that commit.

Reply:
I prefer to remove the unneeded chdir().

Reply:
OK